File: cpp/1020.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int n; cin >> n;

    int anos = n/365;
    int meses = (n%365)/30;
    int dias = (n%365)%30;

    cout << anos << " ano(s)" << "\n";
    cout << meses << " mes(es)" << "\n";
    cout << dias << " dia(s)" << "\n";

}

File: 1020.py
idade_dias=int(input())
ano=idade_dias//365
mes= (idade_dias-(ano*365))//30
dia= (idade_dias -(ano*365)-(mes*30))
print(f"{ano} ano(s)")
print(f"{mes} mes(es)")
print(f"{dia} dia(s)")